.github/workflows/xunit-tests.yml

```diff
@@ -65,6 +65,7 @@ jobs:
       run: dotnet build -warnaserror --no-restore ${{env.solutionPath}}
     - name: Run DafnyLanguageServer Tests
       run: |
+        ## Run twice to catch unstable code (Issue #2673)
```

Comment:
Hmm that leaves me with an aftertaste. Suppose checking in an unstable tests crash 1/4 of the time (3/4 chances of success). After testing twice, we wlll have a success rate of 1/2
If someone then commits the tests when the two runs succeed, confident that a crash would have been caught from the two consecutive runs,
every other subsequent PR will now have 1/2 chances of failing, more than one fourth like the original.
So this is a two-sided coin. By raising the bar to entry, if anything goes wrong randomly, we will affect every one new PR out there.
I'm just saying it out loud as a motivation to claim that
1) We should run the tests only once.
2) If a test fails on any PR, unrelated to that PR, it should be immediately reported as an issue and deactivated so that future PR don't have the same issue that would otherwise accumulate.

Reply:
Up to you (and Remy). I just put in a comment to explain why the line is replicated.